examples/03-advanced/06-stress_gradient_path.py

```diff
@@ -0,0 +1,143 @@
+"""
+.. _stress_gradient_path:
+
+Stress gradient normal to a defined node.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows how to plot a stress gradient normal to a selected node.
```

Comment:
Must the selected node be on a surface? Is so, it might be clearer to specify it.

Reply:
As the code is based on creating a path along the normal, it has to be on the surface to retrieve the normal. Added a comment.